src/main/java/Deadline.java

```diff
@@ -0,0 +1,58 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Name 'by' doesnt sound very descriptive of the variable being stored

Reply:
Thanks! I have changed the name.